apps/chameleon/app/(profile)/profile/edit/page.tsx

```diff
@@ -0,0 +1,182 @@
+              value={'This is your insta name here'}
+              readOnly
+            ></Input>
+          </div>
+          <div className="flex flex-col">
+            <Label className="m-2 w-64 text-lg">
+              I&apos;m currently shooting:
+            </Label>
+            <div className="grid grid-cols-2 md:w-80">
+              {allPhotoShootsTypes.map((photogType) => (
+                <ProfileShootType
+                  key={photogType.id}
+                  isSelected={
+                    userProfileData?.photoShootTypes.find(
+                      (skillName) => skillName.name === photogType.name
+                    )
+                      ? true
+                      : false
+                  }
+                  formatedShootTypeName={formatEnumString(photogType.name)}
+                  photoShootType={{ name: photogType.name }}
+                  user={{ id: user.id }}
+                />
+              ))}
+            </div>{' '}
+          </div>
+          <div className="flex flex-col">
+            <Label className="m-2 w-64 text-lg">I&apos;m open to:</Label>
+            <div className="grid grid-cols-2 sm:grid-cols-3 md:w-80">
+              {allFocusSkills.map((skill) => (
+                <ProfileSkill
+                  key={skill.id}
+                  photographySkill={{
+                    name: skill.name,
+                    skillType: skill.skillType,
+                  }}
+                  formatedSkillName={formatEnumString(skill.name)}
+                  isSelected={
+                    // Since the prisma model PhotographySkill has 2 tables per name, one for "SPECIALTY" and one for "CURRENT_FOCUS",
+                    // this is to make sure the correct name for the skilltype is highlighted when selected
+                    userProfileData?.photographySkills.find(
+                      (profileSkill) =>
+                        profileSkill.name === skill.name &&
+                        profileSkill.skillType === skill.skillType
+                    )
+                      ? true
+                      : false
+                  }
+                  user={{ id: user.id }}
+                />
+              ))}
+            </div>
+            <div className="flex flex-col">
+              <Label className="m-2 w-64 text-lg">
+                I&apos;m most skilled with:
+              </Label>
+              <div className="grid grid-cols-2 sm:grid-cols-3 md:w-80">
+                {allSpecialtySkills.map((skill) => (
+                  <ProfileSkill
+                    key={skill.id}
+                    photographySkill={{
+                      name: skill.name,
+                      skillType: skill.skillType,
+                    }}
+                    formatedSkillName={formatEnumString(skill.name)}
+                    isSelected={
+                      // Since the prisma model PhotographySkill has 2 tables per name, one for "SPECIALTY" and one for "CURRENT_FOCUS",
```

Comment:
code should speak for itself, remove, if the code isn't clear afterwards then it needs to be cleaned

Reply:
I thought the function was readable from an outside perspective but i thought that maybe someone would wonder WHY you would need to find both name and skilltype if someone didn't know that the schema for profileskills had one table for both skill types with the same name

Reply:
if its not 100% clear to you then others will struggle